src/Database/Query/Grammars/RedshiftGrammar.php

```diff
@@ -2,8 +2,21 @@
 namespace YuK1\LaravelRedshift\Database\Query\Grammars;
 
 use Illuminate\Database\Query\Grammars\PostgresGrammar;
+use Illuminate\Database\Query\Builder;
 
 class RedshiftGrammar extends PostgresGrammar
 {
-
-}
\ No newline at end of file
+    /**
+     * Compile insert statement.
+     * Unlike PostgreSQL, in Redshift it is not possible to return the id at the same time
+     *
+     * @param  \Illuminate\Database\Query\Builder  $query
+     * @param  array  $values
+     * @param  string  $sequence
+     * @return string
+     */
+    public function compileInsertGetId(Builder $query, $values, $sequence)
```

Comment:
Is `$sequence` required here?

Reply:
This is the original from the parent class:
![Screenshot 2023-07-14 at 12 30 14](https://github.com/appsfortableau/laravel-redshift/assets/6881597/3822468f-7eed-421b-ac84-41fc6649461c)